Market Maker Guide.md

```diff
@@ -4,14 +4,14 @@ This guide is written for individuals and entities looking to run xud in order t
 
 ## Two Modes
 
-1. **Light setup** using [Neutrino](https://github.com/lightninglabs/neutrino) and a random open eth provider or optionally [Infura](https://infura.io/). This keeps the setup light-weight & cheap, but requires to trust these full nodes delivering correct chain data to a certain extent.
-2. **Full setup** using [bitcoind](https://github.com/bitcoin/bitcoin/), [litecoind](https://github.com/litecoin-project/litecoin) and [geth](https://github.com/ethereum/go-ethereum). Requires more resources and an SSD, but keeps the setup trustless.
+1. **Default: Light setup** using [Neutrino](https://github.com/lightninglabs/neutrino) and a random open eth provider. This keeps the setup light-weight & cheap, but requires somewhat trusting these full nodes to deliver correct chain data.
```

Comment:
some trust from these full nodes?

Reply:
`This keeps the setup light-weight & cheap, but requires somewhat trusting these full nodes to deliver correct chain data.` ->
`This keeps the setup light-weight & cheap, but requires some trust from these full nodes to deliver correct chain data.` ?

Reply:
Maybe then remove the `somewhat trusting` part?

Reply:
Done: https://github.com/ExchangeUnion/docs/pull/104/commits/19bf2237bfe7bd04539971958e8233efceaf688c